mlos_bench/mlos_bench/services/config_persistence.py

```diff
@@ -192,8 +192,8 @@ def build_generic(self, *,
         return inst
 
     def build_environment(self, config: Dict[str, Any],
+                          tunables: TunableGroups,
```

Comment:
Make tunables a non-optional param.

Reply:
Could alternative use the same `*` trick we did earlier to make them all required to be named, but this was fewer changes for now.